### PR TITLE
Response template updates

### PR DESCRIPTION
**Pull request by muselesscreator**
**TL;DR -** Updates the oa_response template for team ORAs based on pilot feedback and figma file.
Joint-PR w/ @nsprenkle 

Screen Shot 2020-09-16 at 13 11 10

**What changed?**
- Added informational text/headings for team assignments.
- Replace "you may continue to work on this assignment..." message with team submission banner for team assignments.
- Bump ORA to `2.9.12`

**Developer Checklist**
- [x] Reviewed the [release process](./release_process.md)
- [x] Translations up to date
- [x] JS minified, SASS compiled
- [x] Test suites passing on Jenkins
- [x] Bumped version number in [setup.py](../setup.py) and [package.json](../package.json)

JIRA:
 [JIRA-5307](https://openedx.atlassian.net/browse/JIRA-5307)
 [JIRA-5308](https://openedx.atlassian.net/browse/JIRA-5308)
 [JIRA-5309](https://openedx.atlassian.net/browse/JIRA-5309)
 [JIRA-5310](https://openedx.atlassian.net/browse/JIRA-5310)

FIY: @edx/masters-devs-gta

### CODE REVIEW

File: openassessment/templates/openassessmentblock/response/oa_response.html [hunk 1]
```diff
@@ -53,16 +62,49 @@ <h4 class="step__title">
         <div class="wrapper--step__content">
             <div class="step__instruction">
                 <p>
+                  {% if team_name %}
+                    {% trans "Enter your team's response to the prompt." %}
+                    {% if submission_due %}
+                      {% blocktrans %}
+                        You can save your progress and return to complete your team's response at any time before the due date
```
Comment (Andytr1) [inline openassessment/templates/openassessmentblock/response/oa_response.html:69]: missing period.

Comment (nsprenkle) [reply openassessment/templates/openassessmentblock/response/oa_response.html:69]: Confusingly, there is a period but it's hiding after the "date" span that's after this.

File: openassessment/templates/openassessmentblock/response/oa_response.html [hunk 2]
```diff
@@ -83,17 +125,51 @@ <h5 class="submission__answer__part__text__title">{% trans "The prompt for this
...
-                                            {% trans "Your response (required)" %}
+                                            {% trans "(Required)" %}
                                           {% elif text_response == "optional" %}
-                                            {% trans "Your response (optional)" %}
+                                            {% trans "(Optional)" %}
                                           {% endif %}
                                         </h5>
+                                        {% if team_name %}
+                                          {% blocktrans %}
+                                              Teams should designate one team member to submit a response on behalf of the
```
Comment (Andytr1) [inline openassessment/templates/openassessmentblock/response/oa_response.html:166]: This comment makes my head hurt...Have we asked UX for a better verbiage if at all possible?

Comment (nsprenkle) [reply openassessment/templates/openassessmentblock/response/oa_response.html:166]: This came from UX and product. But I agree, it is a bit verbose.

File: openassessment/templates/openassessmentblock/response/oa_response.html [hunk 3]
```diff
@@ -53,16 +62,49 @@ <h4 class="step__title">
...
+                    </strong>.
+                  {% else %}
                     {% trans "Enter your response to the prompt." %}
                     {% if submission_due %}
-                    {% trans "You can save your progress and return to complete your response at any time before the due date" %}
-                    (<span class="step__deadline">
-                        <span id="oa_step_deadline_response" class="date ora-datetime" data-datetime="{{ submission_due|timezone:'UTC'|date:'c' }}" data-timezone="{{ user_timezone }}" data-format="longDateTime" data-language="{{ user_language }}"></span>
-                    </span>).
+                      {% blocktrans %}
+                        You can save your progress and return to complete your response at any time before the due date
```
Comment (Andytr1) [inline openassessment/templates/openassessmentblock/response/oa_response.html:92]: Missing period.

File: openassessment/templates/openassessmentblock/response/oa_response.html [hunk 4]
```diff
@@ -53,16 +62,49 @@ <h4 class="step__title">
...
+                          data-language="{{ user_language }}">
+                        </span>
+                      </span>).
+                    {% else %}
+                      {% blocktrans %}
+                        You can save your progress and return to complete your team's response at any time.
+                      {% endblocktrans %}
+                    {% endif %}
+                    <strong class="emphasis">
+                      {% trans "After you submit a response on behalf of your team, it cannot be edited" %}
```
Comment (Andytr1) [inline openassessment/templates/openassessmentblock/response/oa_response.html:86]: Should there be a period at the end?

**Review by Andytr1 — APPROVED** [review]
Did static review. Looks good!

**Comment by jansenk** [comment]
Looks good locally!

**Review by jansenk — APPROVED** [review]
